experimental/dds/tree/src/LogViewer.ts

```diff
@@ -566,7 +594,7 @@ export class CachingLogViewer implements LogViewer {
 	 * @returns Edit information for the earliest known sequenced edit.
 	 */
 	public earliestSequencedEditInSession(): { edit: Edit<ChangeInternal>; sequenceNumber: number } | undefined {
```

Comment:
In general, I think the 'inSession' APIs should be removed. Their purpose was to provide a synchronous API by always ensuring that in session edits are kept in memory. It seems like that might not be the case now so they're no longer needed.

Reply:
Talked to Taylor and understand more about what this PR is trying to do. The `inSession` APIs should be renamed as they don't really have the same meaning anymore but what they are doing is still necessary, just in the context of in memory edits rather than in session edits.